src/Tribe/Importer/Column_Mapper.php

```diff
@@ -88,4 +89,12 @@ private function get_organizer_column_names() {
 			'organizer_phone'   => __( 'Organizer Phone', 'tribe-events-calendar' ),
 		);
 	}
+
+	private function get_additional_fields_column_names() {
+		$additionalFields = tribe_get_option( 'custom-fields' );
+		return array_combine(
+			array_map(function($field) { return $field['name']; }, $additionalFields),
```

Comment:
Same thing here with the Anonymous function, on the first one you should use `wp_list_pluck`.

On the second one, you could `sprintf( __( 'Event %s', 'tribe-events-calendar' ), $field['label'] );` but create a method on the class to do it.